iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -210,19 +207,15 @@ class ConversationListViewController: ViewController,
 
         let conversations = self.conversationListController.conversations
 
-        var snapshot = self.dataSource.snapshot()
+        let snapshot = self.dataSource.updatedSnapshot(with: self.conversationListController)
```

Comment:
Because we need to have consistent behavior for when the load more cell and new conversation cell appear, I moved all that logic to a single function on the data source.